hls4ml/model/graph.py

```diff
@@ -56,8 +56,8 @@ def __init__(self, config):
         else:
             self.writer_config = {
                 'Namespace': None,
-                'WriteWeightsTxt': True,
-                'WriteTar': False,
+                'WriteWeightsTxt': False,
+                'WriteTar': True,
```

Comment:
I think this should be the other way around. The tar is mostly (*) useless, the txt actually speeds up compilation (Csim) a lot so that should remain true by default. Only once you're fully ready and want a clean project to turn into an emulator in CMSSW would you change txt to false.

(*) Mostly because I rely on this feature for the sysnthesis service framework, but I can manage without.

Reply:
Swapped here: 6a48180